cypress/integration/anton.sample/todoSample.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Managing of todo list
+  As a user
+  I want to use todo list functionality
```

Comment:
Better to explain briefly which kind of functionality is offered by todo app: adding, completing, editing, deleting.